src/web/app/pages-instructor/instructor-session-edit-page/instructor-session-edit-page.component.spec.ts

```diff
@@ -700,4 +715,17 @@ describe('InstructorSessionEditPageComponent', () => {
         'The feedback session has been copied. Please modify settings/questions as necessary.',
         { courseid: 'testId2', fsname: 'Test Session' });
   });
+
+  it('should open danger modal if session end time updates end time after any extensions deadline', async () => {
```

Comment:
I don't believe async is needed here. Could you help to check if we can remove async?

Reply:
Yup, no need. My previous implementation awaited something. Thanks